integration-cli/docker_cli_events_test.go

```diff
@@ -230,6 +230,30 @@ func (s *DockerSuite) TestEventsImageUntagDelete(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestEventsImageTag(c *check.C) {
+	time.Sleep(time.Second * 2) // because API has seconds granularity
```

Comment:
Yeah I think this is more granular because let's say `tag` is somehow changed to cause 2 events but that's not the desired behavior. It would be caught here by this test.